lib/Doctrine/ORM/UnitOfWork.php

```diff
@@ -3382,18 +3382,39 @@ private function performCallbackOnCachedPersister(callable $callback)
         }
     }
 
+    private function dispatchFlushEntityListener($event)
+    {
+        foreach ($this->identityMap as $className => $entities) {
+            $class = $this->em->getClassMetadata($className);
+            $invoke = $this->listenersInvoker->getSubscribedSystems($class, $event) & ~ListenersInvoker::INVOKE_MANAGER;
+            if (ListenersInvoker::INVOKE_NONE === $invoke) {
+                continue;
```

Comment:
This is an entity listener, and if there aren't any subscribed, it will just `continue`.